spec/features/user_views_all_users_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+feature "User views all users", %(
+  As a user
+  I want to see a list of users
+  So that I get to know all users
+
+  Acceptance Criteria
+  [x] I must see all users' usernames
+  [x] I must see all users' avatars
+) do
+  before do
+    5.times { FactoryGirl.create(:user) }
+  end
+
+  scenario "user sees other users' usernames" do
+    visit users_path
+
+    expect(page).to have_content(User.first.username)
```

Comment:
These tests are good, though they only make sure that they show the first and last username. 

Consider something like the following: 

```
User.all.each do |u|
  expect(page).to have_content (u.username) 
end
```

That way, you make sure you're hitting all of the users that you created!